README.md

```diff
@@ -4,7 +4,7 @@
 
 This project will copy public DockerHub repositories to a private registry.
 
-It's possible to filter by docker tags, tag age and number of latest tags.
+It's possible to filter by docker tags, tag age and number of latest tags, it is also possible to use a private docker registry.
```

Comment:
Can  you add an example for how this is supposed to work? What option do I specify and how does it impact the process?

Reply:
Added an example, let me know what you think!